src/Domain/BlockLogicalCoordinates.cpp

```diff
@@ -151,7 +171,7 @@ std::vector<block_logical_coord_holder<Dim>> block_logical_coordinates(
       const functions_of_time_type& functions_of_time);
 
 GENERATE_INSTANTIATIONS(INSTANTIATE, (1, 2, 3),
-                        (::Frame::Inertial, ::Frame::Grid))
+                        (::Frame::Inertial, ::Frame::Grid, ::Frame::Distorted))
```

Comment:
(Nitpick) let's try to order frames in lists like this "grid, distorted, inertial", i.e. the order in which they are composed